Week1/homework/app.js

```diff
@@ -1,11 +1,169 @@
 'use strict';
 
 {
+  // Step 1.1
   const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets'
+    'to_kill_a_mockingbird',
+    '1984',
+    'harry_potter_and_the_philosophers_stone',
+    'the_lord_of_the_rings',
+    'the_great_gatsby',
+    'pride_and_prejudice',
+    'the_diary_of_a_young_girl',
+    'the_book_thief',
+    'the_hobbit',
+    'little_women'
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // Step 1.3
+  // function generateUlAndLiWithIds() {
+  //   const ul = document.createElement('ul');
+  //   for (let i = 0; i < bookTitles.length; i++) {
+  //     const li = document.createElement('li');
+  //     ul.appendChild(li);
+  //     li.setAttribute('id', bookTitles[i]);
+  //     console.log(li);
+  //   }
+  // }
+  // console.log(generateUlAndLiWithIds());
```

Comment:
In the homework assignment it was intended for this function to be used. Then in the second function from step 1.8 you were to update the elements created with the images for each book. They way you have done it works too, of course (and perhaps even preferred), but it not completely following the assignment.

As a general note, you should leave commented out code in a pull request. It is of no interest to the reviewer.